catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -681,12 +675,9 @@ private Metacard getRegistryMetacardFromRegistryPackage(RegistryPackageType regi
         Metacard metacard;
 
         try {
-            JAXBElement<RegistryPackageType> jaxbRegistryObjectType =
-                    RIM_FACTORY.createRegistryPackage(registryPackage);
-
-            ByteArrayOutputStream baos = new ByteArrayOutputStream();
-            parser.marshal(marshalConfigurator, jaxbRegistryObjectType, baos);
-            InputStream xmlInputStream = new ByteArrayInputStream(baos.toByteArray());
+            InputStream xmlInputStream =
```

Comment:
Should we consider adding a method that returns an `InputStream` to the "metacard helper" class since we're doing this a few times?